pkgs/tools/text/ocrmypdf/default.nix

```diff
@@ -0,0 +1,103 @@
+{ fetchFromGitHub
+, ghostscript
+, img2pdf
+, jbig2enc
+, leptonica
+, pngquant
+, python3
+, python3Packages
+, qpdf
+, stdenv
+, tesseract4
+, unpaper
+}:
+
+let
+  inherit (python3Packages) buildPythonApplication;
+
+  runtimeDeps = with python3Packages; [
+    ghostscript
+    jbig2enc
+    leptonica
+    pngquant
+    qpdf
+    tesseract4
+    unpaper
+    pillow
+  ];
+
+in buildPythonApplication rec {
+  pname = "ocrmypdf";
+  version = "8.2.3";
+  disabled = ! python3Packages.isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "jbarlow83";
+    repo = "OCRmyPDF";
+    rev = "v${version}";
+    sha256 = "1ldlyhxkav34y9d7g2kx3d4p26c2b82vnwi0ywnfynb16sav36d5";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    pytest
+    pytestrunner
+    setuptools
+    setuptools_scm
+    setuptools-scm-git-archive
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    cffi
+    chardet
+    img2pdf
+    pdfminer
+    pikepdf
+    reportlab
+    ruffus
+  ] ++ runtimeDeps;
+
+  checkInputs = with python3Packages; [
+    hocr-tools
+    pypdf2
+    pytest
+    pytest-helpers-namespace
+    pytest_xdist
+    pytestcov
+    pytestrunner
+    python-xmp-toolkit
+    setuptools
+  ] ++ runtimeDeps;
```

Comment:
`runTimeDeps` is already added to `propagatedBuildInputs`, so no need to add here again

Reply:
Removed it there. I had to add it to checkInputs to make the tests work when they are enabled. Is that expected?

Reply:
If those are only needed for tests, then yes. If they are needed when using this program, then they should be added to `(propagated)(buildInputs)`.

Reply:
I think it's the former. It seems to work how it is now. The only thing I know about at this point is I haven't ever been able to get it this version to work in nix-review but it seems to work when installed. Not sure if that's an issue?

Reply:
Action needed here?